docs/kyma/10-09-backup.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Cannot create a volume snapshot
+type: Troubleshooting
+---
+
+If a PersistentVolumeClaim is not bound to a PersistentVolume, the attempt to create a volume snapshot from that PersistentVolumeClaim will fail with no retries. An event will be logged to indicate no binding between the PersistentVolumeClaim and the PersistentVolume.
+
+This may happen if PersistentVolumeClaim and VolumeSnapshot specifications are in the same YAML file. As a result, the VolumeSnapshot object is and the PersistentVolumeClaim object are created at the same time, but the PersisitentVolume is not available yet and cannot be bound to the PersistentVolumeClaim. 
```

Comment:
```suggestion
This may happen if PersistentVolumeClaim and VolumeSnapshot specifications are in the same YAML file. As a result, the VolumeSnapshot and the PersistentVolumeClaim objects are created at the same time, but the PersisitentVolume is not available yet so it cannot be bound to the PersistentVolumeClaim. 
```

Reply:
what happened here :O